package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hackathon2020",
-  "version": "3.1.0",
+  "version": "3.2.0",
```

Comment:
I think this should be 4.0.0 already

Reply:
why? I didnt do any breaking changes

Reply:
Yeah, ok, true. I thought because it's almost completely a new version, but I guess you merged most of the things before this MR anyway